inginious/frontend/submission_manager.py

```diff
@@ -321,15 +321,15 @@ def add_job(self, task, inputdata, debug=False):
 
         return submissionid, to_remove
 
-    def _delete_exceeding_submissions(self, username, task, max_submissions_bound=-1):
+    def _delete_exceeding_submissions(self, username, task, task_dispenser, max_submissions_bound=-1):
```

Comment:
Either there is a call to this function that has not been updated, either the ``max_submissions_bound`` argument is never used and removed from the signature.